app/controllers/bookings_controller.rb

```diff
@@ -1,3 +1,29 @@
 class BookingsController < ApplicationController
   #TODO index, show, new, create
+
+  def index
+    @bookings = Booking.all
+  end
+
+  def show
+    @Booking = Booking.find(params[:id])
```

Comment:
should be lowercase @booking

Reply:
Maybe we will have to wait til tomorrow to figure out how to change things